### DISCUSSION TITLE
Make an egui desktop app stay on top

### DISCUSSION THREAD

**Discussion by couleurm** [emilk/egui#4366] [Q&A]
I'm not talking about a new window inside egui being stay on top, but the actual window on your operating system

How does one go about toggling on/off such OS feature?

**Comment by YgorSouza** [comment]
At startup: [ViewportBuilder::window_level](https://docs.rs/egui/latest/egui/viewport/struct.ViewportBuilder.html#structfield.window_level). At any time: [ViewportCommand::WindowLevel](https://docs.rs/egui/latest/egui/viewport/enum.ViewportCommand.html#variant.WindowLevel).

**Comment by couleurm** [comment]
works, thank you :)
```rs
ctx.send_viewport_cmd(egui::ViewportCommand::WindowLevel(egui::WindowLevel::Normal));
ctx.send_viewport_cmd(egui::ViewportCommand::WindowLevel(egui::WindowLevel::AlwaysOnTop));
```
